CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Adding new features, improving documentation, fixing bugs, writing new tests,
 designing and coding new examples or writing tutorials are all examples of 
 helpful contributions.
 
-## Prerequisits
+## Prerequisities
```

Comment:
Thanks for the fix, but I think this is also the wrong spelling... See one line below.

Reply:
> Thanks for the fix, but I think this is also the wrong spelling... See one line below.

Right, changed to [prerequisites](https://www.merriam-webster.com/dictionary/prerequisites).